docs/change-notes/changelist/v2-changes.md

```diff
@@ -2,37 +2,43 @@
 
 Below is a summary of changes that have been made in Gaffer version 2.
 
+Each breaking change has been put into one of 3 tiers, where they are also relevant to the tiers below:
+
+- **user**: end users might have to change their Gaffer queries
+- **admin**: Gaffer admins that manage graphs might have to change their config or deployment environment
+- **developer**: developers of Gaffer or Gaffer extensions might need to make code changes
+
 ### Accumulo 2 Support
 The Accumulo store now supports Accumulo 2 and Hadoop 3 by default, with support for Accumulo 1 and Hadoop 2 retained. See the [Accumulo Migration page](../migrating-from-v1-to-v2/accumulo-migration.md) for more information about this change.
 
 ### Federated Store Improvements
 The Federated Operation was added to greatly improve flexibility of using a Federated Store.
-!!! danger "Breaking change"
+!!! danger "Breaking change | user | [alpha-0.4](https://github.com/gchq/Gaffer/releases/tag/gaffer2-2.0.0-alpha-0.4)"
     To migrate, please see the [Federated Store Changes page](../migrating-from-v1-to-v2/federation-changes.md).
 
 ### Cache Improvements and fixes
 All "caches" within Gaffer received a lot of bug fixes which should make them significantly more stable and consistent over time. This should improve usability of FederatedStores, NamedOperations and NamedViews.
-!!! danger "Breaking change"
+!!! danger "Breaking change | admin | [alpha-0.6](https://github.com/gchq/Gaffer/releases/tag/gaffer2-2.0.0-alpha-0.6)"
     The cache will need to be reloaded, as the new internal cache interface has changed.  To do this, export all of the contents of your cache, upgrade, then re-add everything manually.
 
 ### Removal of Deprecated code
 All of Gaffer 1's deprecated code has been removed.
-!!! danger "Breaking change"
+!!! danger "Breaking change | user | [alpha-0.1](https://github.com/gchq/Gaffer/releases/tag/gaffer2-2.0.0-alpha-0.1)"
```

Comment:
Shouldn't this be "developer" or both of these?

Reply:
I tried to make it like a hierarchy, where "user" actually means "user, admin and developer". I tried to describe it at the top but I don't think it's very clear.

Reply:
Have explained it clearly, what do you think.

Reply:
It's clearer now.